docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -45,18 +45,26 @@ Following are the steps involved in your TechPass onboarding journey.
 
   <kbd>![accept-invitation](assets/images/onboarding/po-non-se/accept-invitation.png)</kbd>
 
+  !> Your TechPass account is already created by this time but will in pending status until you accept this invitation. If you do not accept this invitation, this pending account will be deleted.
```

Comment:
can add the following: "... will be deleted after 30 days."

Reply:
done. Thank you!